Dockerfile

```
@@ -59,5 +59,7 @@ RUN etcd --version
 
 FROM scratch
 ARG ETCD_UNSUPPORTED_ARCH
+LABEL org.opencontainers.image.url="https://hub.docker.com/r/rancher/hardened-etcd"
+LABEL org.opencontainers.image.source="https://github.com/rancher/image-build-etcd.git"
```

1. don't set the image URL; we push this to multiple locations
2. just link to the project without the .git suffix.
```suggestion
LABEL org.opencontainers.image.source="https://github.com/rancher/image-build-etcd"
```